packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImport-moved.md

```diff
@@ -2,7 +2,9 @@
 
 In order to support multiple chart libraries, Victory based charts have moved. This rule will update import paths to our victory directory.
 
-Additionally, Victory has become a peer dependency.
+Additionally, Victory is now an optional peer dependency, allowing future chart libraries to be installed without including Victory dependencies and vice versa
```

Comment:
@kmcfaul @edonehoo Do we need this information in the upgrade section for people upgrading to v6?